web/profiles/custom/os2loop/modules/os2loop_flag_content/src/Form/FlagContentForm.php

```diff
@@ -125,8 +126,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#required' => TRUE,
     ];
 
-    $form['actions']['#type'] = 'actions';
-    $form['actions']['submit'] = [
+    $form['submit'] = [
```

Comment:
Please wrap the action buttons in https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21Actions.php/class/Actions/9.1.x.